pkg/api/scylla/v1/cluster_types.go

```diff
@@ -80,6 +80,9 @@ type ScyllaClusterSpec struct {
 	// When Scylla Manager is not installed, these will be ignored.
 	// +optional
 	Backups []BackupTaskSpec `json:"backups,omitempty"`
+	// ImagePullSecrets for Scylla and Agent images.
```

Comment:
nit: `imagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling Scylla and Agent images.`

Reply:
done